Content.Shared/Damage/Components/DamageableComponent.cs

```diff
@@ -401,17 +434,17 @@ public void OnExplosion(ExplosionEventArgs eventArgs)
                 _ => throw new ArgumentOutOfRangeException()
             };
 
-            ChangeDamage(DamageType.Piercing, damage, false);
-            ChangeDamage(DamageType.Heat, damage, false);
+            ChangeDamage(GetDamageType("Piercing"), damage, false);
+            ChangeDamage(GetDamageType("Heat"), damage, false);
         }
     }
 
     [Serializable, NetSerializable]
     public class DamageableComponentState : ComponentState
     {
-        public readonly Dictionary<DamageType, int> DamageList;
+        public readonly Dictionary<DamageTypePrototype, int> DamageList;
```

Comment:
I don't think prototypes can be directly sent over the network, the string ids should be sent instead and resolved client-side.